### PR TITLE
Validation bug prof-20

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**
To allow only alpha characters in first, last and middleName fields

### CODE REVIEW

File: app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js [hunk 1]
```diff
@@ -73,17 +80,15 @@ export default class RaceAndEthnicity extends React.Component {
       this.state.childClient.primary_race === 'Unable to determine*'
     ) {
       return '';
-    } else if (ethnicities < 1) {
+    } else if (ethnicities.length < 1) {
```
Comment (kelfink) [inline app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: I don't see this being unit tested.  Especially needed if we're fixing a bug.  Show that an empty list of ethnicities returns this string.

File: app/javascript/Clients/clientIdPage/ClientInformation/ClientInformation.js [hunk 2]
```diff
@@ -60,19 +60,19 @@ export default class ClientInformation extends React.Component {
     let errorMessage = this.state.errorMessage;
     switch (fieldName) {
       case 'common_first_name':
-        const firstNameValid = /^\w*[\w\d]$/i.test(value);
+        const firstNameValid = /^[A-Za-z\s]+$/i.test(value);
```
[the remark this answers is not in the document]
Comment (Sushmareddy-OSI) [reply app/javascript/Clients/clientIdPage/ClientInformation/ClientInformation.js]: R-05437 is only for CIN field, For first, middle and last name fields the rule says to accept only alpha characters. 
R - 04966 Names must have at least one alpha character - There must be at least one alpha character in at least one of the name fields (first, middle, last).

[the remark this answers is not in the document]
Comment (kelfink) [reply app/javascript/Clients/clientIdPage/ClientInformation/ClientInformation.js]: Ok.  I put in a clarifying question on the story for R-04966 to make sure this intends to restrict us to alpha-only names.  which, in my mind would have been a simpler way to word it.  "Alpha only"

[the remark this answers is not in the document]
Comment (kelfink) [reply app/javascript/Clients/clientIdPage/ClientInformation/ClientInformation.js]: I think we got the clarify.  Alphanumerics should be allowed, meaning it needs to allow [A-Za-z0-9]+, or, if we enforce Alpha first, then /[A-Z]. [A-Za-z0-9]*$/

[the remark this answers is not in the document]
Comment (kelfink) [reply app/javascript/Clients/clientIdPage/ClientInformation/ClientInformation.js]: If we enforce "at least one alpha" and also "only alphanumeric". then I believe this regex does it:  
/(?=.*[a-zA-Z])([a-zA-Z0-9]+)$/

**Pull request merged**